lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -2343,47 +2343,71 @@ input[name="subject"] {
         }
 
       }
-      .arrow {
-        font-weight: bold;
-      }
+        .arrow {
+            background: $white;
+            border: 1px solid $uxpl-blue-base;
+            color: $uxpl-blue-base;
+            font-weight: bold;
+        }
     }
   }
 
 }
 
 // Custom File upload
 .customBrowseBtn {
-  margin: ($baseline/2) 0;
-  display: inline-block;
-  .file-browse {
+    display: inline-block;
     position: relative;
+    margin: ($baseline/2) 0;
+    width: 350px;
     overflow: hidden;
-    display: inline;
-    @include margin-left(-5px);
-    span.browse {
-      @include button(simple, $uxpl-blue-base);
-      @include margin-right($baseline);
-      padding: 6px ($baseline/2);
-      font-size: 12px;
-      border-radius: 0 3px 3px 0;
+    vertical-align: bottom;
+  
+    #browseFile,
+    #bulk-exception-upload {
```

Comment:
I disagree that this is beyond the scope of the PR. If you are adding CSS selectors they should be classes, not ids. What is the level of effort to add a class to the DOM where these ids are?